repose-aggregator/commons/commons-utilities/src/main/scala/org/openrepose/commons/utils/servlet/http/HttpServletResponseWrapper.scala

```diff
@@ -99,6 +102,48 @@ class HttpServletResponseWrapper(originalResponse: HttpServletResponse, headerMo
   def this(originalResponse: HttpServletResponse, headerMode: ResponseMode, bodyMode: ResponseMode) =
     this(originalResponse, headerMode, bodyMode, originalResponse.getOutputStream)
 
+  override def setStatus(i: Int, s: String): Unit = {
+    message = s
+    super.setStatus(i, s)
```

Comment:
According to the servlet 3.1 specification, "Headers set after the response is committed will be ignored by the servlet container."

This seems to imply that headers _can_ be set after the response is committed, they just won't be sent in the response. That is how the code is written now, so I'll leave it that way for the time being.

Also, "The sendError method will set the appropriate headers and content body for an
error message to return to the client. An optional String argument can be provided
to the sendError method which can be used in the content body of the error." That's basically all that the Servlet 3.1 specification says about `sendError(...)`. I still wonder if all containers set the HTTP status message, or if that's just something Jetty does. I'm not certain if we should set the HTTP status message or just the content body. For the moment, I've chained `sendError(i, s)` through to `setStatus(i, s)` so that we'll get the behavior of the container (although the `s` parameter of those methods does not necessarily represent the same thing).